core-services/prow/02_config/_config.yaml

```diff
@@ -196,6 +196,12 @@ plank:
   - config:
       timeout: 6h0m0s
     repo: openshift/osde2e
+  - config:
+      timeout: 3h0m0s
```

Comment:
The default timeout for this is 4h, you are reducing it

Reply:
In that case we need to tweak a different timeout. @alvaroaleman, can you guide us here?
Here's an example of  what we need to fix: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-logging-operator/1169/pull-ci-openshift-cluster-logging-operator-release-5.2-e2e-gcp/1437765835715252224 died saying
```
 {"component":"entrypoint","file":"prow/entrypoint/run.go:165","func":"k8s.io/test-infra/prow/entrypoint.Options.ExecuteProcess","level":"error","msg":"Process did not finish before 2h0m0s timeout","severity":"error","time":"2021-09-14T15:42:55Z"} 
```

Reply:
https://docs.ci.openshift.org/docs/architecture/timeouts/#step-registry-test-process-timeouts

Reply:
I was going by this document https://docs.ci.openshift.org/docs/architecture/timeouts/#prow-test-process-timeouts